server/server_test.go

```diff
@@ -31,7 +31,10 @@ func TestMain(m *testing.M) {
 	SetupLogging(config)
 	mlog.Info("Loaded config", mlog.String("filename", configFile))
 
-	s = New(config)
+	s, err = New(config)
+	if err != nil {
+		mlog.Err(err)
```

Comment:
An error here might get overlooked. I'm leaning towards a more strict check that does just exit if `err != nil`.

Reply:
Agreed!

Reply:
Should the test also `panic` here?
```suggestion
		panic(err)
```